frontend/packages/dev-console/integration-tests/features/addFlow/create-from-git.feature

```diff
@@ -0,0 +1,174 @@
+Feature: Create Application from git form
+   As a user, I want to create the application, component or service from Add options
+
+   Background:
+      Given user is at developer perspective
+      And user is at Add page
+      And user has created namespace starts with "aut-addflow-git"
+
+
+   @smoke
+   Scenario Outline: Add new git workload with new application for resoruce type "<resource_type>" : A-04-TC02, A-04-TC13
+      Given user is at Import from git page
+      When user enters Git Repo url as "https://github.com/sclorg/dancer-ex.git"
+      And user enters Application name as "dancer-ex-git-app"
+      And user enters Name as "<name>"
+      And user selects resource type as "<resource_type>"
+      And user clicks Create button on Add page
+      Then user will be redirected to Topology page
+      And user is able to see workload "<name>" in topology page
+
+      Examples:
+         | name            | resource_type     |
+         | dancer-ex-git   | Deployment        |
+         | dancer-ex-git-1 | Deployment Config |
+
+
+   @regression
+   Scenario: Add new git workload to the existing application : A-04-TC03
+      Given user created workload "nodejs-ex-git" with resource type "Deployment"
+      And user is at Add page
+      And user is at Import from git page
+      When user enters Git Repo url as "https://github.com/sclorg/dancer-ex.git"
+      And user enters Application name as "nodejs-ex-git-app"
+      And user enters Name as "dancer-ex.git"
+      And user selects resource type as "Deployment Config"
+      And user clicks Create button on Add page
+      Then user will be redirected to Topology page
+      And created workload "dancer-ex.git" is linked to existing application "nodejs-ex-git-app"
+
+
+   @regression
+   Scenario: Cancel the git workload creation : A-04-TC04
+      Given user is at Import from git page
+      When user enters Git Repo url as "https://github.com/sclorg/dancer-ex.git"
+      And user clicks Cancel button on Add page
+      Then user will be redirected to Add page
+
+
+   @regression
+   Scenario: Create workload without application route : A-04-TC05
+      Given user is at Import from git page
+      When user enters Git Repo url as "https://github.com/sclorg/dancer-ex.git"
+      And user enters Application name as "app-no-route"
+      And user enters Name as "name-no-route"
+      And user unselects the advanced option Create a route to the application
+      And user clicks Create button on Add page
+      Then user will be redirected to Topology page
+      And public url is not created for node "name-no-route"
```

Comment:
Do we need to have it like the route decorator not present on the node "name-no-route"

Reply:
Yes, because we need to verify whether route is created or not

Reply:
```suggestion
      And public url is not created for node "name-no-route" in the workload sidebar
```